app/validators/return-versions/setup/additional-submission-options.validator.js

```diff
@@ -15,25 +15,48 @@ const Joi = require('joi')
  * If these requirements are not met the validation will return an error.
  *
  * @param {object} options - The options extracted from payload taken from the request to be validated
+ * @param {object} session - The session instance
  *
  * @returns {object} The result from calling Joi's schema.validate(). If any errors are found the `error:` property will
  * also exist detailing what the issue is.
  */
-function go(options) {
+function go(options, session) {
   const additionalSubmissionOptions = options.additionalSubmissionOptions
 
   const errorMessage = 'Select additional submission options for the requirements for returns'
 
   const schema = Joi.object({
     additionalSubmissionOptions: Joi.array().items(Joi.string()).required()
-  }).messages({
-    'any.required': errorMessage,
-    'array.sparse': errorMessage
   })
+    .custom((value, helpers) => {
+      return _noQuarterlyReturnsForSummerCycle(value, helpers, session)
+    }, 'No quarterly returns for summer cycle')
+    .messages({
+      'any.required': errorMessage,
+      'array.sparse': errorMessage,
+      'any.invalid': "Quarterly returns submissions can't be set for returns in the summer cycle"
```

Comment:
The [GOV.UK style guide](https://www.gov.uk/guidance/style-guide/a-to-z) tells us to avoid negative contractions.

```suggestion
      'any.invalid': "Quarterly returns submissions cannot be set for returns requirements in the summer cycle"
```